airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/SchemaUpdateNotifications.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { FormattedMessage } from "react-intl";
+import { useAsyncFn } from "react-use";
+
+import { ControlLabels } from "components";
+import { Card } from "components/ui/Card";
+import { Switch } from "components/ui/Switch";
+
+import { useConnectionEditService } from "hooks/services/ConnectionEdit/ConnectionEditService";
+
+import styles from "./SchemaUpdateNotifications.module.scss";
+
+export const SchemaUpdateNotifications: React.FC = () => {
+  const { connection, updateConnection, connectionUpdating } = useConnectionEditService();
+  const [{ loading }, onChange] = useAsyncFn(async (checked: boolean) => {
+    await updateConnection({
```

Comment:
@krishnaglick not sure about this one, but this just popped into my mind. do you think we should handle errors here? in case of failed request what will happen here?

Reply:
Sounds good! I added toast notifications for the failed request